packages/models/src/lib/persist.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from 'zod';
+
+/**
+ * Define Zod schema for the PersistConfig type
+ *
+ * @example
+ *
+ * // Example data for the UploadConfig type
+ * const uploadConfigData = {
+ *   server: 'https://example.com/api',
+ *   apiKey: 'your-api-key',
+ * };
```

Comment:
Again, not an accurate example, should be persist config instead of upload config.

Reply:
fixed